content/blog/20241129_fluentd-v1.18.0-has-been-released.md

```diff
@@ -0,0 +1,141 @@
+# Fluentd v1.18.0 has been released
+
+Hi users!
+
+We have released v1.18.0 on 2024-11-29. ChangeLog is [here](https://github.com/fluent/fluentd/blob/master/CHANGELOG.md#release-v1180---20241129).
+
+This release is a new release of v1.18 series.
+In this release, we added some new features and fixed bugs.
+
+## Enhancement
+
+### Add zero-downtime-restart feature
+
+In this release, we have added the feature to update the Fluentd process with zero downtime for Linux and macOS environments (not supported on Windows).
+Fluentd had a feature called by graceful reload that reloads the configuration without restart.
+However, the graceful reload feature has a problem that causes a downtime, it cannot receive the data from network using `in_udp`, `in_tcp`, and `in_syslog` plugins during reloading configuration.
+
+The zero-downtime-restart feature has been introduced to solve the problem of the graceful reload.
+This feature supports Fluentd upgrades and configuration reloads with zero downtime.
+
+This feature can be used by sending a `SIGUSR2` signal to the supervisor process,
+and this changes behavior `SIGUSR2` from graceful reload to zero-downtime-restart feature on Linux and macOS environments.
+
+Please see [#4624](https://github.com/fluent/fluentd/pull/4624) for more information.
+
+### Add with-source-only feature
+
+This feature has been added to support the zero-downtime-restart feature that it launches Fluentd with input plugins only.
+
+This feature has added `--with-source-only` option in `fluentd` command, `with_source_only` option in system configuration and `out_buffer` plugin which can be used for buffering and relabeling events.
+These are used internally to support zero-downtime-restart.
+
+Please see [#4661](https://github.com/fluent/fluentd/pull/4661) for more information.
+
+### Config File Syntax: Extend Embedded Ruby Code support for Hashes and Arrays
+
+The [Embedded Ruby Code](https://docs.fluentd.org/configuration/config-file#embedded-ruby-code) has been supported in configuration.
+In this release, we have extended the syntax to allow the use of Embedded Ruby Code in Hashes and Arrays.
+
+Example:
+
+```
+key1 ["foo","#{1 + 1}"] # Embedded Ruby Code in Array syntax
+key2 {"foo":"#{1 + 1}"} # Embedded Ruby Code in Hash syntax
+```
+
+Result:
+
+```
+key1 ["foo","2"]
+key2 {"foo":"2"}
+```
+
+Please note that this is not backward compatible, although we assume that this will never affect to actual existing configs.
+
+In case the behavior changes unintentionally, you can disable this feature by surrounding the entire value with single quotes.
+
+Example:
+
+```
+key '{"foo":"#{1 + 1}"}'
+```
+
+### transport tls: Use `SSL_VERIFY_NONE` by default
+
+We have changed the flags to `SSL_VERIFY_NONE` by default for server the certification verification.
```

Comment:
It might better to explain "why" a bit.
When client_cert_auth is false (default), VERIFY_PEER causes troublesome, so it was changed to SSL_VERIFY_NONE.

Reply:
https://github.com/fluent/fluentd-website/pull/335/commits/3fc04afccd1991c9d568999603be1e314baf2817